### PR TITLE
Update to v2.0.4, templates, and add permissions

### PR DESCRIPTION
**Pull request by tituspijean**
Update PR and issue templates
Update to v1.3.5
Add `nodes` permission

### CODE REVIEW

**Comment by tituspijean** [comment]
aaaand my CI is broken.
!testme

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -111,11 +125,14 @@ ynh_system_user_create --username=$app --home_dir=$final_path
 #=================================================
 ynh_script_progression --message="Installing Node-RED..." --weight=30
 
-chown -R $app: $final_path
+chown -R root:root $final_path
```
Comment (alexAubin) [inline scripts/upgrade:128]: Uuuuh I get very suspicious when I see this ... If root is the only owner, then that means any other user that needs to enter $final_path goes through the "others" permission (which also means any other user on the system can enter) - *or* the app runs as root

But maybe it doesn't matter for this app or there's a good reason for this :s

Comment (tituspijean) [reply scripts/upgrade:128]: I have actually no good explanation of why I put this line... Let's revert it.

**Comment by tituspijean** [comment]
[Test Badge](https://ci.pijean.ovh/ci/job/4)

**Comment by tituspijean** [comment]
[Test Badge](https://ci.pijean.ovh/ci/job/5)

**Comment by alexAubin** [comment]
Sounds okay but for some reason the change_url test is broken now :thinking:

**Comment by tituspijean** [comment]
[Test Badge](https://ci.pijean.ovh/ci/job/22)

File: scripts/install [hunk 2]
```diff
@@ -148,15 +153,22 @@ yunohost service add $app --description="Low-code programming for event-driven a
...
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start"
 
 #=================================================
 # SETUP SSOWAT
 #=================================================
 ynh_script_progression --message="Configuring permissions..." --weight=1
 
-ynh_permission_update --permission="main" --remove="all_users" --add="$admin"
-ynh_permission_create --permission="ui" --url="/ui" --allowed="$admin"
```
Comment (alexAubin) [inline scripts/install:159]: Eeeeh, should we add something in the upgrade script to delete this legacy permission ?

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
[Test Badge](https://ci.pijean.ovh/ci/job/26)

**Pull request merged**